core/cmd/local_client_vrf_test.go

```diff
@@ -0,0 +1,233 @@
+package cmd_test
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/internal/mocks"
+	"github.com/smartcontractkit/chainlink/core/store/models/vrfkey"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+)
+
+const (
+	vrfPasswordFilePath = "./testdata/vrf_password.txt"
+	vrfKeyFilePath      = "./testdata/vrf_key.json"
+	// This is the public key found in the vrk key file
+	vrfPublicKey = "0xe2c659dd73ded1663c0caf02304aac5ccd247047b3993d273a8920bba0402f4d01"
+)
+
+func TestLocalClientVRF_ListVRFKeys(t *testing.T) {
+	t.Parallel()
+
+	store, cleanup := cltest.NewStore(t)
+	t.Cleanup(cleanup)
```

Comment:
R A R E :tada: 
A
R
E
:tada:

Reply:
We should probably use this instead of defer...